src/sentry/relay/config/metric_extraction.py

```diff
@@ -317,8 +317,8 @@ def _is_widget_query_low_cardinality(widget_query: DashboardWidgetQuery, project
     if query_killswitch:
         return False
 
-    if not widget_query.columns:
-        # No columns means no high-cardinality tags.
+    # No columns or only errors means no high-cardinality tags.
+    if not widget_query.columns or widget_query.conditions == "conditions=event.type:error":
```

Comment:
I think this should be whether the `conditions` string contains `event.type:error`?